components/integration/production/kustomization.yaml

```diff
@@ -13,3 +14,11 @@ namespace: integration-service
 
 components:
   - ../k-components/manager-resources-patch
+
+patches:
+  - path: pipelines-as-code-secret-patch.yaml
```

Comment:
This doesn't seem to match the filename of the patch `pipelines-as-code-secret.yaml` that's also added in this PR.

Reply:
Yes.....Thanks!